docs/layouts.md

```diff
@@ -2,7 +2,7 @@
 
 Layout components are used to wrap pages. Layouts should contain components like headers, footers or sidebars that will be used across the site.
 
-Layouts are _just_  **.vue components** located in `src/layouts` and need to be [declared as a global](#make-a-layout-global) component or imported per page to be used.
+Layouts are _just_ &nbsp;**.vue components** located in `src/layouts` and need to be [declared as a global](#make-a-layout-global) component or imported per page to be used.
```

Comment:
It looks like this is adding a double space between 'just' and '.vue'?

Reply:
I take that back - I see what this is addressing now. It looks like GitHub is processing the markdown slightly differently than Gridsome is. 

Good catch!

Reply:
It is 100% for readability.  Without the `&nbsp;` the docs render as:

![image](https://user-images.githubusercontent.com/1228996/74183937-7df85780-4c0b-11ea-8484-2248ef5bb736.png)

With the added `&nbsp;` they render as:

![image](https://user-images.githubusercontent.com/1228996/74184086-be57d580-4c0b-11ea-81cd-bc64c78cc316.png)